src/ZODB/tests/testConnection.py

```diff
@@ -521,29 +529,18 @@ def doctest_invalidate(self):
         Transaction ids are 8-byte strings, just like oids; p64() will
         create one from an int.
 
-        >>> cn.invalidate(p64(1), {p1._p_oid: 1})
+        >>> mvcc_storage.invalidate(p64(1), {p1._p_oid: 1})
```

Comment:
I agree they didn't make sense on Connection. I was just afraid that some app, probably Zope2 or Plone, with my luck, would be relying on them. But if you're comfortable removing them so am I.